### ISSUE TITLE
Add support for .well-known file/access

### ISSUE THREAD

**Issue by duglin** [xregistry/spec#120]
See: https://github.com/Fannon/xregistry-brainstorming/blob/main/.well-known/xregistry.json

Split into a few steps:
1 - define our .well-known file
2 - define how the data/files section - to allow for downloading of the xRegistry metadata

@Fannon

**Comment by Fannon** [comment]
One idea: The `.well-known/xregistry.json` could be an alias, pointing to the root of the xRegistry API (if there is one). 
But the root of the xRegistry cannot be assumed to be in `/` of the host. Especially I would recommend to have the major version of the API protocol in there, so I would expect that a typical xRegistry API base path would be like `/xregistry/v1/`.

**Comment by Fannon** [comment]
@duglin , @deissnerk : Here is a first PR: https://github.com/xregistry/spec/pull/125

**Comment by Fannon** [comment]
Related: #106

**Comment by duglin** [comment]
@Fannon should we close this one?

**Comment by Fannon** [comment]
> @Fannon should we close this one?

yes, I think we agreed in Frankfurt that we will not have this and just link to everything relevant from xRegistry base (root entry point)
